freqtrade/optimize/hyperopt/hyperopt_interface.py

```diff
@@ -44,10 +44,11 @@ def __init__(self, config: Config) -> None:
     def generate_estimator(self, dimensions: list[Dimension], **kwargs) -> EstimatorType:
         """
         Return base_estimator.
-        Can be any of "GP", "RF", "ET", "GBRT" or an instance of a class
-        inheriting from RegressorMixin (from sklearn).
+        Can be any of "TPESampler", "GPSampler", "CmaEsSampler", "NSGAIISampler"
+        "NSGAIIISampler", "QMCSampler" or an instance of a class
+        inheriting from BaseSampler (from optuna.samplers).
         """
-        return "ET"
+        return "NSGAIISampler"
```

Comment:
As a reminder (to myself and you) 
Similar "things" will need to be updated in the docs ...

Reply:
done